packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -368,4 +359,11 @@ export class ApexTestNode extends TestNode {
   public contextValue = 'apexTest';
 }
 
-export const testOutlineProvider = new ApexTestOutlineProvider(null);
```

Comment:
So this was leading to all kinds of sadness trying to both write the tests and ensure when this ApexTestOutputProvider was being constructed.  I don't love the poor substitute for a singleton, but I would have had to majorly refactor the int tests if I altered how this property is referenced. 

This seemed like the lesser of two evils path but I'm open to suggestions.

Reply:
This looks like a solid solution to avoid the big refactor to me.